.github/workflows/apply.yml

```diff
@@ -27,5 +27,5 @@ jobs:
       - name: terraform apply
         uses: dflook/terraform-apply@v1
         with:
-          path: newrelic/synthetics
+          path: .
```

Comment:
Strange that [this is required](https://github.com/dflook/terraform-apply#inputs).

Reply:
Ah, I've been meaning to make that feature request https://github.com/dflook/terraform-github-actions/issues/78